Description: Move plugins dir
Author: <marc.cluet@ubuntu.com>
Upstream: no

--- a/etc/client.cfg.dist
+++ b/etc/client.cfg.dist
@@ -1,6 +1,6 @@
 main_collective = mcollective
 collectives = mcollective
-libdir = /usr/libexec/mcollective
+libdir = /usr/share/mcollective/plugins
 logger_type = console
 loglevel = warn
 
--- a/etc/server.cfg.dist
+++ b/etc/server.cfg.dist
@@ -1,6 +1,6 @@
 main_collective = mcollective
 collectives = mcollective
-libdir = /usr/libexec/mcollective
+libdir = /usr/share/mcollective/plugins
 logfile = /var/log/mcollective.log
 loglevel = info
 daemonize = 1
